openbb_terminal/miscellaneous/library/trail_map.csv

```diff
@@ -398,7 +399,7 @@ stocks.fa.fama_raw,openbb_terminal.stocks.fundamental_analysis.dcf_model.get_fam
 stocks.fa.historical_5,openbb_terminal.stocks.fundamental_analysis.dcf_model.get_historical_5,
 stocks.fa.similar_dfs,openbb_terminal.stocks.fundamental_analysis.dcf_model.get_similar_dfs,
 stocks.fa.analysis,openbb_terminal.stocks.fundamental_analysis.eclect_us_model.get_filings_analysis,
-stocks.fa.dcf,openbb_terminal.stocks.fundamental_analysis.fmp_model.get_dcf,
+stocks.fa.dcfc,openbb_terminal.stocks.fundamental_analysis.fmp_model.get_dcf,
```

Comment:
@jmaslek was this a typo? 
If not, it will cause this line `hist_dcf = openbb.stocks.fa.dcf(symbol)` to break on `equity.ipynb`.
Also, `website/content/sdk/guides/intros/stocks/fundamental-analysis.md` and `website/content/sdk/reference/stocks/fa/dcf.md` would need to change - although I assume this is now auto generated.

Reply:
Is intentional. `dcf` in terminal runs dcf in excel.  I can edit the nb to reflect that.  And yes documentation is autogenerated.